closed/autoconf/custom-spec.gmk.in

```diff
@@ -118,3 +118,9 @@ OPENSSL_BUNDLE_LIB_PATH:=@OPENSSL_BUNDLE_LIB_PATH@
 OPENSSL_CFLAGS:=@OPENSSL_CFLAGS@
 OPENSSL_DIR:=@OPENSSL_DIR@
 WITH_OPENSSL:=@WITH_OPENSSL@
+
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+  OPENJ9_VM_BUILD_DIR := $(OUTPUT_ROOT)/vm/runtime
+else
+  OPENJ9_VM_BUILD_DIR := $(OUTPUT_ROOT)/vm
+endif
```

Comment:
This doesn't look right: `OUTPUT_ROOT` is used in jdk8, but in jdk11+ it should be `OUTPUTDIR`.